tests/integration/test_transcribe.py

```diff
@@ -23,6 +26,21 @@ def transcribe_snapshot_transformer(snapshot):
     reason="Vosk transcription library has issues running on Circle CI arm64 executors.",
 )
 class TestTranscribe:
+    @staticmethod
+    def _wait_transcription_job(client, transcribe_job_name):
```

Comment:
The return type of this function is currently inconsistent ('None' or 'True')

Also add typehints.